Src/spinlex.c

```diff
@@ -1667,6 +1731,8 @@ lex(void)
 	{	switch (c) {
 		case '-': c = follow('>', IMPLIES,    '-'); break;
 		case '[': c = follow(']', ALWAYS,     '['); break;
+		case '/': c = follow('\\', AND, '/'); break;
+		case '\\': c = follow('/', OR, '\\'); break;
```

Comment:
What is the purpose of this change?

Reply:
nimble called this particular commit as "inline ltl operation abbreviations" ...